src/pages/posts/ListingPosts.vue

```diff
@@ -0,0 +1,322 @@
+<template>
+  <div class="q-pa-lg">
+    <div class="row justify-between">
+      <q-input color="grey-9" v-model="text" label="Pesquisar" rounded outlined>
+        <template v-slot:append>
+          <q-icon name="search" />
+        </template>
+      </q-input>
+      <q-btn size="15px" flat round color="grey-9" icon="add">
+        <q-tooltip v-model="showing">Novo Post</q-tooltip>
+      </q-btn>
+    </div>
+    <div class="q-pa-md row justify-center">
+      <card>
```

Comment:
percebi q vc está dando copy paste nessa estrutura card. Vc poderia ter uma computada q seria um array de objetos, e utilizar um v-for

exemplo:

```
computed: {
 cardList () {
   return [
    { 
      image: 'https://cdn.quasar.dev/img/parallax2.jpg' 
      category: 'Categoria',
      title: 'Titulo 1',
      content: 'Lorem...',
      author: 'Cleyton',
      createdAt:  '...',
     ....
    },
    ...
  ]
 }
}

 <card v-for="(card, index) in cardList" :key="index">
        <template v-slot:image>
          <div class="col-6">
            <q-img class="card__image" :src="card.image">
              <div class="absolute-bottom text-subtitle1 text-center">
                {{ card.category }}
              </div>
            </q-img>
          </div>
        </template>
        <template v-slot:text>
          <h1 class="text-h4">{{ card.title }}</h1>
          <span class="sub-title q-mb-md text-body1 ellipsis-3-lines">
             {{ card.content }}
          </span>
        </template>
      ....

```